### ISSUE TITLE
Unable to use glyph

### ISSUE THREAD

**Issue by happyalu** [huertatipografica/sahitya#3]
I'm a fan of Alegreya, and was utterly delighted to see Sahitya. The font looks great. I'm trying to use it in libreoffice, and I notice that for the unicode sequence U+092C U+094D U+0930 U+0942, I am getting incorrect glyph. It seems that the font does contain a special glyph for uni092C094D0930, which is the correct shape, but I am unable to use it. What is missing?

By the way, the same problem exists for many other chars involving U+094D U+0930. 

![bru](https://cloud.githubusercontent.com/assets/231523/11952536/169ae682-a8bf-11e5-967e-5b85bf76cd7f.png)

**Comment by davelab6** [comment]
Does this work on the web?

**Comment by happyalu** [comment]
Tried typing ब्रू in http://typecast.com/preview/google/Sahitya on both Firefox and Chrome (I'm on Linux) and both show the same incorrect character as Libreoffice.

**Comment by davelab6** [comment]
Sounds like its a font problem, not a libreoffice problem, then :) Thanks!

**Comment by davelab6** [comment]
> What is missing?

Probably the OpenType GSUB logic is buggy.

**Comment by happyalu** [comment]
I'm not a really a fonts guy, but I'd be happy to help. :-)

**Comment by happyalu** [comment]
Not sure if this helps at all, but "Glyph Info" in fontforge for the glyph in Sahitya (left) versus Laila (Right) is attached. 
![sahitya-laila](https://cloud.githubusercontent.com/assets/231523/11953147/0270d6c6-a8c4-11e5-9c71-dd835e82eea2.png)

**Comment by davelab6** [comment]
Right, this is showing you the OpenType GSUB Subtables relevant to that
glyph.

BTW, looks like you have a very old version of FontForge installed, or just
not the latest theme.

**Comment by happyalu** [comment]
Wow. Version 20120731!! That's what apt-get got me :-) Had just installed to try it out today for the first time.

**Comment by davelab6** [comment]
I think there is a ppa

**Comment by happyalu** [comment]
Might have to compile from source -- will try to do it later. (PPA seems not to have added Ubuntu 15.10 yet). Also, not exactly sure what I'll be doing once I have fontforge working :-)

I tried editing the ligature entry with old fontforge to make it look like what it was in Sahitya, exported the font and installed it, but that didn't seem to fix anything.

**Comment by juandelperal** [comment]
Thank you very much @happyalu for pointing this. I will fix it.
Did you find another bug in the font?

**Comment by happyalu** [comment]
Thanks, @juandelperal , will look forward to the fix. This is a really good font!

I haven't used this font very much yet, but a quick typing test highlighted two more errors:
![sahityamore](https://cloud.githubusercontent.com/assets/231523/11954929/c3d63046-a8d2-11e5-9a98-dbd2b3228a09.png)

**Comment by juandelperal** [comment]
Thanks! I will take a look there too. Please type directly the text: it is better to me so I can copy and paste in the browser :)

**Comment by happyalu** [comment]
ब्र
ब्रू
कॉं
त्र्य

**Comment by juandelperal** [comment]
Better! Take a look now

**Comment by happyalu** [comment]
Edit: Sorry, most of this was fixed. My font cache wasn't updated. Please see my comment below on another bug.

Thanks, @juandelperal . It seems the issue is partially fixed. I'm basing this on libreoffice. Here's the actual text, and screenshot. The first row is now fixed! But the second and third still have issues.

ब्र ब्रे ब्रै ब्रो ब्रि ब्री 
ब्रु ब्रू

कॉं त्र्य

![sahitya-1](https://cloud.githubusercontent.com/assets/231523/11957761/9f974398-a8e9-11e5-8797-1a3295442de3.png)

**Comment by happyalu** [comment]
Another issue: is this related to kerning, maybe?
Text is: ब्द

![bd](https://cloud.githubusercontent.com/assets/231523/11958022/4f1114ec-a8eb-11e5-99e3-d6a03bba09ae.png)

**Comment by happyalu** [comment]
Thanks, @juandelperal , ब्र is now fixed, So is त्र्य and कॉं. 

ब्रू still needs to be fixed:

![bru](https://cloud.githubusercontent.com/assets/231523/11958598/b85f8674-a8ee-11e5-83ca-6bad843ff09e.png)

**Comment by juandelperal** [comment]
All fixed! Please reinstall now the otfs files
![captura de pantalla 2015-12-22 a la s 12 39 33](https://cloud.githubusercontent.com/assets/2414972/11958912/5769ed5e-a8a9-11e5-8463-0a0153a82d81.png)

**Comment by happyalu** [comment]
ब्र ब्द त्र्य कॉं ब्रू

Everything except the last one seems to be working for me. But your image shows it working. Not sure what may be going on.

![bru](https://cloud.githubusercontent.com/assets/231523/11959271/6934265a-a8f2-11e5-9f8e-3945f529c05a.png)

**Comment by happyalu** [comment]
Sorry, @juandelperal ; just noticed a couple more things; the first three seem to have issue with distance between half forms.

त्व त्त्व    व्य   ब्रू

![tva](https://cloud.githubusercontent.com/assets/231523/11971045/99285376-a95c-11e5-9091-431fad0e081a.png)

**Comment by juandelperal** [comment]
Thanks @happyalu. This project started in Glyphs 1, and then migrated to Glyphs 2. I think the naming conventions for the automatic openType features generation has changed, that's why there are many bugs.
I opened this public Google doc, so anyone can add conjuncts to fix. Please add whatever you find there:
https://docs.google.com/document/d/1v17a2arvo08TCeTioqS4lzjaMwT_TWxl8ZfgDTjJFlk/edit

I will make a deeper review soon. Thanks again
j.

**Comment by happyalu** [comment]
Thanks. I added a few more to that doc. 
Thanks also for looking at this so quickly! :-)

**Comment by davelab6** [comment]
Vesper libre is a good model

**Comment by happyalu** [comment]
Any update :-)
